KomaMRIFiles/src/Phantom/Phantom.jl

```diff
@@ -167,7 +168,8 @@ end
 function export_motion!(motion_group::HDF5.Group, motion::ArbitraryMotion)
     HDF5.attributes(motion_group)["model"] = "ArbitraryMotion"
     HDF5.attributes(motion_group)["K"] = size(motion.dx)[2] + 1
```

Comment:
Why do we need this?

Reply:
This was intended to add redundancy to the file and to be able to check that the information is correct. Again, if you don't like it now, I will remove it.

Reply:
Yes, please remove it.

Reply:
Solved in [e10474b](https://github.com/JuliaHealth/KomaMRI.jl/pull/408/commits/e10474b6f40c7118821a0da02b709a706d9e8509)